test/process_wrapper/rustc_quit_on_rmeta.rs

```diff
@@ -68,8 +68,7 @@ mod test {
         ]);
         assert!(
             !out_content.contains("should not be in output"),
-            "output should not contain 'should not be in output' but did: {}",
-            out_content
+            "output should not contain 'should not be in output' but did",
```

Comment:
I believe this change is related to https://github.com/rust-lang/rust-clippy/pull/10055

Unfortunately `#[allow(clippy::uninlined_format_args]` cannot be used here without requiring we raise our min supported Rust version above 1.59.0

Reply:
Can `rustversion::attr` be used here? https://crates.io/crates/rustversion

(I don't know the answer - I know `rustversion` can be used for conditional attrs on structs, and can't be used for conditional enabling of features - I'm not sure how `allow`s fit in :))

Reply:
I don't think it'd be appropriate to add a new external dependency just for this. It seems it'll be fixed in a future nightly release so I think it's an acceptable trade off for now. If I can remember it I'll add them back in the future